content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,214 @@
+=================
+Track & Bill Time
```

Comment:
No ampersands, please.
```
Track and bill time
```

Reply:
@StraubCreative For the record, the ampersands were not a creative flourish, I included them to match the UI 😂
Should I just leave them out of headings, or out of the doc entirely?

Reply:
Ah good q. Honestly I think it might be one of the tricky areas of the style you use where you map out the UI with headings (as opposed to general business case for heading and use guilabels in body content). Could go either way with this one, your call 😜

Reply:
I spoke with @samueljlieber and he confirmed that they won't cause any rst issues, so I am going to remove them from headings, but will leave in any direct ui references just to keep consistent 👍